bc_obps/registration/tests/models/test_operation.py

```diff
@@ -252,17 +252,6 @@ def test_change_status_then_assign_id_succeeds(self):
         self.non_registered_operation.save()
         self.assertIsNotNone(self.non_registered_operation.bcghg_id)
 
-    def test_prevent_status_change_from_registered_with_assigned_ids(self):
-        # Ensure changing status from 'REGISTERED' prevents updates while retaining IDs
-        self.registered_operation.bcghg_id = baker.make_recipe("registration.tests.utils.bcghg_id")
-        self.registered_operation.bc_obps_regulated_operation = baker.make_recipe("registration.tests.utils.boro_id")
-        self.registered_operation.save()
-
-        self.registered_operation.status = Operation.Statuses.DRAFT
-        with self.assertRaises(ProgrammingError):
-            self.registered_operation.save()
-        self.assertIsNotNone(self.registered_operation.bcghg_id)
-
```

Comment:
deleted this test because it was failing and as far as we're aware at the moment, there won't ever be a scenario where an operation's status reverts from "Registered" back to "Draft"